next.config.js

```diff
@@ -29,8 +29,17 @@ const nextConfig = {
         destination: `${process.env.NEXT_PUBLIC_BACK_URL}/users/sign_out`,
       },
       {
-        source: '/api-docs/:path*',
-        destination: `${process.env.NEXT_PUBLIC_BACK_URL}/api-docs/:path*`,
+        source: '/users/:path*',
+        destination: `${process.env.NEXT_PUBLIC_BACK_URL}/users/:path*`,
+      },
+      {
+        source: '/users/auth/google_oauth2/callback',
+        destination: `${process.env.NEXT_PUBLIC_BACK_URL}
+        /users/auth/google_oauth2/callback`,
```

Comment:
This route doesn't exist yet in the backend, therefore return an error page when trying to access.